neon_homeassistant_skill/locale/en-us/intents/disable.intent

```diff
@@ -0,0 +1,2 @@
+Disable Home Assistant
+I don't use Home Assistant
```

Comment:
after normalization this would be  "i do not use home assistant". Depending on STT I think you may also get "i dont use home assistant"

Reply:
This does still work for me, I think because the second utterance is the original un-normalized string

Reply:
Worth adding the second one? I've been wary of too many Padatious alternatives lately

Reply:
Right, maybe not with Padatious since it does its own fuzzy matching